documentation/modules/rn-2.4.adoc

```diff
@@ -146,3 +150,11 @@ Previously, the `cluster-admin` role was required to browse and create providers
 .Convert i440fx to q35
 
 Migration of virtual machines with i440fx chipset is now supported. The chipset is converted to q35 during the migration. link:https://issues.redhat.com/browse/MTV-430[(MTV-430)]
+
+.Preserve the UUID setting in SMBIOS for a VM that is migrated from RHV
+
+The Universal Unique ID (UUID) number within the System Management BIOS (SMBIOS) doesn't change anymore for VMs that are migrated from RHV. This fix enables applications that run within the guest operating system and depend on this setting (e.g., for licensing) to operate on the target OCP cluster the same way as they did on RHV. link:https://issues.redhat.com/browse/MTV-597[(MTV-597)]
+
+.Do not expose password for RHV in error messages
+
+Previously, the password that was specified for ovirt-engine appeared in error messages that were displayed in the web console and logs when faling to connect to RHV. In this release, error messages that are generated when failing to connect to RHV do not reveal the password for ovirt-engine.
```

Comment:
Is "ovirt-engine" an attribute or the same for upstream and downstream? 
"...that were displayed in the web console and logs when MTV failed to connect to RHV. In this release, error messages that are generated when MTV fails to connect to RHV do not reveal the password for ovirt-engine. "

Use attributes for MTV and RHV.

Reply:
> @ahadas Is "ovirt-engine" an attribute or the same for upstream and downstream?

well, that's a good point - the underlying component is the same (ovirt-engine) but we better call it RHVM on downstream here

Reply:
commented on #398